governance/polls/Management of Velodrome Airdrop and veNFT - June 13, 2022.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Management of Velodrome Airdrop and veNFT - June 13, 2022
+summary: Signal your support or opposition to incentivizing DAI LP pairs on Velodrome.
+discussion_link: https://forum.makerdao.com/t/signal-request-incoming-velodrome-airdrop-on-optimism-velo-venft/15371
+vote_type: Plurality Voting
+categories:
+   - Medium Impact
+   - Misc Governance
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2022-06-13T16:00:00
+end_date: 2022-06-16T16:00:00
+---
+# Poll: Management of Velodrome Airdrop and veNFT - June 13, 2022
+
+The Governance Facilitators have placed a Governance Poll into the voting system on behalf of the Maker Community. This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for three days beginning on Monday, June 13 at 16:00 UTC.
+
+**This is a binary vote.** 
+- **You may vote for a single option.** 
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to the following changes: 
+* Incentivize the DAI<>USDC and DAI<>OP pairs on Velodrome.
+* Send the resulting trading fees to the MakerDAO Optimism Governance Address.
+
+[For context](https://twitter.com/VelodromeFi/status/1529549901239050240), MakerDAO will be receiving an airdrop totaling ~1% of voting power to direct emissions on Velodrome.
+
+Please review the discussion [thread](https://forum.makerdao.com/t/signal-request-incoming-velodrome-airdrop-on-optimism-velo-venft/15371) to help inform your position before voting.
+
+## Outcomes
+
+**If the votes for the 'Yes' option exceed the votes for the 'No' option then the following actions will be taken:**
```

Comment:
Do we have a Governance Address yet? If so, include it above in the Review section, if not, I'd include setting it up here as an outcome as we will need it set-up to enact the outcome of this poll

Reply:
We do not at the moment :) adding